data/cloudcraft.json

```diff
@@ -0,0 +1,1768 @@
+                  "example": "keep-alive"
+                }
+              },
+              "Content-Encoding": {
+                "schema": {
+                  "type": "string",
+                  "example": "gzip"
+                }
+              },
+              "Date": {
+                "schema": {
+                  "type": "string",
+                  "example": "Wed, 14 Nov 2018 20:56:59 GMT"
+                }
+              },
+              "ETag": {
+                "schema": {
+                  "type": "string",
+                  "example": "c6ed6253ec05e79c94809947df59dfed"
+                }
+              },
+              "Transfer-Encoding": {
+                "schema": {
+                  "type": "string",
+                  "example": "chunked"
+                }
+              },
+              "Vary": {
+                "schema": {
+                  "type": "string",
+                  "example": "Accept-Encoding"
+                }
+              },
+              "X-Content-Type-Options": {
+                "schema": {
+                  "type": "string",
+                  "example": "nosniff"
+                }
+              },
+              "X-Frame-Options": {
+                "schema": {
+                  "type": "string",
+                  "example": "deny"
+                }
+              },
+              "X-XSS-Protection": {
+                "schema": {
+                  "type": "string",
+                  "example": "1; mode=block"
+                }
+              }
+            },
+            "content": {
+              "text/plain": {
+                "examples": {
+                  "Retrieve blueprint": {
+                    "value": "{\n    \"id\": \"bp37712a-c507-4c62-ad8b-7d981cacb3be\",\n    \"data\": { See Blueprint data format specification },\n    \"createdAt\": \"2022-01-01T20:54:47.302Z\",\n    \"updatedAt\": \"2022-01-01T20:55:52.876Z\",\n    \"CreatorId\": \"us46e9aa-5806-4cd6-8e78-c22d58602d09\",\n    \"LastUserId\": \"us46e9aa-5806-4cd6-8e78-c22d58602d09\"\n}"
+                  }
+                }
+              }
+            }
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "Blueprints"
+        ],
+        "summary": "Update a blueprint",
+        "description": "Update an existing blueprint.\n\nThe body of the request should contain the updated blueprint document in JSON format.\n\nOptionally, a conditional update of the blueprint can be perfomed by including the _If-Match_ HTTP header with the same ETag value as provided by the \"Retrieve blueprint\" API. If the blueprint has been modified since the retrieval, the update will be rejected with a _412 Resource out of date_ response. If the update succeeds, the new ETag will be returned.\n\nResponses\n=========\n- **204 No Content on success**\n- 401 Unauthorized\n- 403 Forbidden, insufficient privileges\n- 404 Blueprint not found\n- 412 Resource out of date",
```

Comment:
```suggestion
        "description": "Update an existing blueprint.\n\nThe body of the request should contain the updated blueprint document in JSON format.\n\nOptionally, a conditional update of the blueprint can be perfomed by including the _If-Match_ HTTP header with the same ETag value as provided by the \"Retrieve blueprint\" API. If the blueprint has been modified since the retrieval, the update is rejected with a _412 Resource out of date_ response. If the update succeeds, the new ETag is returned.\n\nResponses\n=========\n- **204 No Content on success**\n- 401 Unauthorized\n- 403 Forbidden, insufficient privileges\n- 404 Blueprint not found\n- 412 Resource out of date",
```